lib/components/common/StyledTable.tsx

```diff
@@ -5,36 +5,19 @@ import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them